timelapse/notify.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 
-def notify(title, text):
+def notify(title: str, text: str) -> None:
```

Comment:
Since we only support macOS, I think it's safe to assume that the return type will be `int`. At least according to the docs.

Reply:
Ahh didnt see a return statement so I assumed it was 'None'. Thanks! Fixed in https://github.com/mre/timelapse/pull/33/commits/2785fc745c4056739d3864ba0b680b3655163019